src/newgrf_config.cpp

```diff
@@ -313,6 +313,10 @@ void GRFParameterInfo::SetValue(struct GRFConfig *config, uint32 value)
 void GRFParameterInfo::Finalize()
 {
 	this->complete_labels = true;
+	if (this->min_value > this->max_value) {
+		grfmsg(2, "StaticGRFInfo: incoherent min/max values, forcing default value");
+		this->min_value = this->max_value = this->def_value;
```

Comment:
is it correct to just force a default value, or should the grf be rejected as invalid? I'd be tempted to say the latter...

Reply:
what would the game d with an "invalid" GRF, and how would people debug their grf?

Reply:
The specs say that a min value for parameters is by default 0 is and a max value of 2^32-1 (which are also the hard limits); default value is to be assumed 0, if not specified otherwise in action 14.